ResourceServer/Models/TrueHomeContext.cs

```diff
@@ -215,16 +237,19 @@ public static async Task<int> createApartment(Apartment ap)
             return id;
         }
         //Delete Apartment
-        public static void deleteApartment(int id)
+        public static bool deleteApartment(int? id)
         {
+            bool isSuccess = false;
             query = "DELETE FROM Apartment" +
                     $" WHERE id_ap = {id};";
 
             using (var connection = new NpgsqlConnection(AppSettingProvider.connString))
             {
                 connection.Open();
                 connection.Execute(query);
+                isSuccess = true;
```

Comment:
isSuccess bez sensu - jeśli query wykona się dobrze to funkcja zwróci true, a jeśli nie to rzuci Exception, które nie jest łapane więc zwróci 500 internal server error, a nie false. Jeśli już to zrób blok try/catch i w catch isSuccess=false. Inna sprawa że query może też nie wyrzucić błędu tylko po prostu nic nie usunąć bo będą złe parametry a wtedy też błędnie zwrócisz true.
DELETE zwraca liczbę usuniętych rekordów, więc lepiej sprawdzać czy zwróciło 1 i na tej podstawie ocenić czy isSuccess czy nie.